docs/source/tutorials/building_blocks.rst

```diff
@@ -334,11 +334,11 @@ Scenes
 ******
 
 The :class:`.Scene` class is the connective tissue of manim.  Every mobject has
-to be :meth:`~.Scene.add` -ed to a scene to be displayed, or
-:meth:`~.Scene.remove` -d from it to cease being displayed.  Every animation
-has to be :meth:`~.Scene.play` -ed by a scene, and every time interval where no
+to be :meth:`added <.Scene.add>` to a scene to be displayed, or :meth:`removed
+<~.Scene.remove>` from it to cease being displayed.  Every animation has to be
+:meth:`played <~.Scene.play>` by a scene, and every time interval where no
```

Comment:
When using custom text for the links, `~` destroys the links (because `~` is only used to shorten the automatically generated text).

```suggestion
<.Scene.remove>` from it to cease being displayed.  Every animation has to be
:meth:`played <.Scene.play>` by a scene, and every time interval where no
```

Reply:
Good call!